src/k_retrieval.py

```diff
@@ -0,0 +1,31 @@
+def k_retrieval(distances_list, similarity_methods, k):
+    
+    """
+    
+    Returns the K more similar images of the database from a list of distance obj
+    
+    """
+    
+    method=distances_list[0].method
```

Comment:
why is it a list and not a single distance?

Reply:
I've assumed a list of distances as input, so I think it's easier to rank the distance. Do you think would be better do it different?